src/L1SharedBridge.sol

```diff
@@ -313,36 +196,25 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
         whenNotPaused
         returns (L2TransactionRequestTwoBridgesInner memory request)
     {
-        require(l2BridgeAddress[_chainId] != address(0), "ShB l2 bridge not deployed");
-
-        (address _l1Token, uint256 _depositAmount, address _l2Receiver) = abi.decode(
-            _data,
-            (address, uint256, address)
-        );
-        require(_l1Token != L1_WETH_TOKEN, "ShB: WETH deposit not supported");
-        require(BRIDGE_HUB.baseToken(_chainId) != _l1Token, "ShB: baseToken deposit not supported");
+        require(l2BridgeAddress[_chainId] != address(0), "USDC-ShB l2 bridge not deployed");
 
-        uint256 amount;
-        if (_l1Token == ETH_TOKEN_ADDRESS) {
-            amount = msg.value;
-            require(_depositAmount == 0, "ShB wrong withdraw amount");
-        } else {
-            require(msg.value == 0, "ShB m.v > 0 for BH d.it 2");
-            amount = _depositAmount;
+        (address _l1Token, uint256 _depositAmount, address _l2Receiver) = abi.decode(_data, (address, uint256, address));
+        require(_l1Token == L1_USDC_TOKEN, "USDC-ShB: Only USDC deposits supported");
```

Comment:
you could put the constant everywhere, but if you want a smaller diff then this is probably better